alot/db/message.py

```diff
@@ -107,13 +107,11 @@ def get_thread_id(self):
         return self._thread_id
 
     def get_message_parts(self):
-        """returns a list of all body parts of this message"""
+        """yield all body parts of this message"""
         # TODO really needed? email  iterators can do this
```

Comment:
Sorry I don't understand how this iterator can be used. It seems to me that it can only return parts of a specific type but our if-else dance checks for several different content types. So we would have to iterate over the email several times?